libcxxabi/include/cxxabi.h

```diff
@@ -77,6 +77,9 @@ extern _LIBCXXABI_FUNC_VIS void __cxa_end_cleanup();
 #endif
 extern _LIBCXXABI_FUNC_VIS std::type_info *__cxa_current_exception_type();
 
+// GNU extension
+extern _LIBCXXABI_FUNC_VIS _LIBCXXABI_NORETURN void __cxa_call_terminate(void*) throw();
+
```

Comment:
We should also file an issue against Clang to ask them to start using `__cxa_call_terminate` instead of `__clang_call_terminate` when the deployment target supports it. Or at least make them aware of this possibility.